src/sagemaker/jumpstart/types.py

```diff
@@ -1103,14 +913,61 @@ def __init__(
         self.hub_content_name = hub_content_name
         self.hub_content_version = hub_content_version
 
+    @staticmethod
+    def extract_region_from_arn(arn: str) -> Optional[str]:
+        """Extracts hub_name, content_name, and content_version from a HubContentArn"""
+
+        HUB_CONTENT_ARN_REGEX = (
+            r"arn:(.*?):sagemaker:(.*?):(.*?):hub-content/(.*?)/(.*?)/(.*?)/(.*?)$"
+        )
+        HUB_ARN_REGEX = r"arn:(.*?):sagemaker:(.*?):(.*?):hub/(.*?)$"
+
+        match = re.match(HUB_CONTENT_ARN_REGEX, arn)
+        hub_region = None
+        if match:
+            hub_region = match.group(2)
+
+            return hub_region
+
+        match = re.match(HUB_ARN_REGEX, arn)
+        if match:
+            hub_region = match.group(2)
+            return hub_region
+
+        return hub_region
+
+
+class CreateHubResponse(JumpStartDataHolderType):
```

Comment:
consider making another module called `interfaces.py` or something like that, i feel like these data holder classes for the Hub APIs belong somewhere different than the other data types for PySDK constructs.